.github/workflows/scanner-update-nvd.yaml

```diff
@@ -0,0 +1,41 @@
+name: Fetch and Update NVD CVEs
+
+on:
+  push:
+    branches:
+      - yli3/**
+  schedule:
+    - cron: '0 0 * * *'  # this runs every day at midnight
+
+jobs:
+  fetch-cves:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set up JQ
+      run: sudo apt-get install jq
```

Comment:
The runner already comes with `jq` installed: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md. 
```suggestion
```